maps/geminus_ophelia/geminus_neue-2.dmm

```diff
@@ -180,8 +180,8 @@
 "adx" = (/obj/structure/railing{dir = 4; icon_state = "railing0"; tag = "icon-railing0 (EAST)"},/obj/structure/railing,/turf/simulated/floor/carpet/gaycarpet,/area/bridge_hallway)
 "ady" = (/obj/structure/table/bench/padded,/obj/machinery/light/colored{dir = 4; icon_state = "yellow1"; tag = "icon-yellow1 (EAST)"},/turf/simulated/floor/tiled/dark,/area/crew_quarters/kitchen)
 "adz" = (/obj/structure/painting/random,/turf/simulated/floor/wood/mahogany,/area/outpost/research/anomaly)
-"adA" = (/obj/machinery/light/colored{dir = 4; icon_state = "yellow1"; tag = "icon-yellow1 (EAST)"},/turf/simulated/floor/carpet/purcarpet,/area/crew_quarters/fitness)
-"adB" = (/obj/structure/disposalpipe/segment{dir = 1; icon_state = "pipe-c"},/obj/machinery/light/colored,/turf/simulated/floor/tiled/dark,/area/crew_quarters/fitness)
+"adA" = (/obj/structure/prop/water_fountain{icon_state = "water_fountain"; dir = 8; step_x = 11},/turf/simulated/floor/bluegeo,/area/planets/Geminus/indoor/shoppingarea)
```

Comment:
The `step_x` and `step_y` vars are used for custom movement systems and must never be used. What you meant to use was the `pixel_x` and `pixel_y` vars. This is one of seven such occurrences.